cherrypicks.go

```diff
@@ -133,11 +190,6 @@ func suggestCherryPicks(
 	}
 
 	// suggest cherry-picking with a comment
-	// nolint:lll
```

Comment:
Still moved :)

Reply:
this is to disable the line length linter for the line I had to move in the first place :))

Reply:
I see, it is because you are adding to the string the `apiWarning`.

Never mind. I missed it